ansible/roles/config-idm-server/tasks/configure_idm.yml

```diff
@@ -0,0 +1,40 @@
+---
+- name: Setup hostname entry
+  lineinfile:
+    dest: /etc/hosts
+    line: "{{ private_ip }} {{ idm_dns_name }}"
+    state: present
+     
+- name: Configure initial IdM setup
+  command: >
+    ipa-server-install -U
+      --hostname="{{ idm_dns_name | default(ansible_fqdn) }}"
+      --domain="{{ idm_domain }}"
+      --realm="{{ idm_realm | upper }}"
+      --ds-password="{{ idm_dm_password }}"
+      --admin-password="{{ idm_admin_password }}"
+      --no-host-dns
+  ignore_errors: true
```

Comment:
Why `ignore_errors` ?

Reply:
For inexplicable reasons, ipa-server-install will sometimes report (falsely) that it had errors when in fact it installed successfully.  Also, the installer is not idempotent and will fail on multiple runs.

Reply:
Is there a bugzilla we can refer to ?

Reply:
I notice the ipa-server role (that I'm reconciling) does the same thing.  I think the biggest problem is idempotency here.  I do not recall the exact message that you receive and haven't been able to reproduce recently when it "fails." So I'm not able to point to a particular "bug."